package.json

```diff
@@ -113,14 +114,14 @@
         "@js-soft/eslint-config-ts": "1.6.9",
         "@js-soft/license-check": "1.0.9",
         "@nmshd/connector-sdk": "*",
-        "@nmshd/content": "5.4.1",
-        "@nmshd/core-types": "5.4.1",
+        "@nmshd/content": "5.5.0",
```

Comment:
this has to be bumped in the sdk too

please run `ncu -i -ws` in monorepos to not miss the sdk.